docs/modules/data-structures/pages/vector-search-overview.adoc

```diff
@@ -235,4 +236,45 @@ To decrease pressure on heap memory, you can decrease the number of parallel mig
 
 1. For searches with small `topK` (for example, 10) it may be beneficial to artificially increase `topK`, adjust `partitionLimit` accordingly, and discard extra results. If you need 10 results, a good starting point for tuning could be `topK=100` and a `partitionLimit` between 50 and 100. While this will make the search slower, it will also improve quality, sometimes significantly. Overall, this setup can be more efficient than increasing index build parameters (`max-degree`, `ef-construction`) which results in slower index builds and searches. With a very small `topK` or `paritionLimit`, the search algorithm is less able to escape local minima and find the best results.
 2. Vector deduplication does not incur significant overhead for uploads (usually less than 1%) and searches. You may consider disabling it to get slightly better performance and smaller memory usage if your dataset does not contain duplicated vectors. However, be aware that in the presence of many duplicated vectors with deduplication disabled, a  similarity search may return poor quality results.
-
+3. For given query each vector index partition is searched by 1 thread. Number of concurrent partition searches is configured by specifying pool size for `hz:query` executor, which by default has 16 threads per member. In configurations with many cores it can be increased to fully utilize all available cores as follows:
```

Comment:
```suggestion
3. For a given query, each vector index partition is searched by 1 thread. The number of concurrent partition searches is configured by specifying a pool size for `hz:query` executor, which by default has 16 threads per member. In configurations with many cores, it can be increased to fully utilize all available cores as follows:
```

Reply:
Does TPC affect or change this guidance at all?

Reply:
no, vector collection search will work in the same way with TPC. Search is not executed on partition thread. OTOH gets and puts should be able to benefit from direct routing to correct partition thread.